torch/onnx/symbolic_opset11.py

```diff
@@ -390,6 +390,12 @@ def sort(g, self, dim, decending, out=None):
     return sym_help._sort_helper(g, self, dim, decending=decending, out=out)
 
 
+@parse_args("v", "i", "i")
```

Comment:
Would it be possible to write a symbolic for previous opsets?

Reply:
In addition to opset11, symbolic was added to opset9 and opset10.